src/routes/demos/activate/.Step5MicrosoftIntegration.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+  export let tenantId;
+  export let clientId;
+  export let privateKey;
+  export let clientCertificate;
+
+  const logo = 'https://cdn.jsdelivr.net/npm/simple-icons@v7/icons/microsoft.svg';
+  const privateKeyPlaceholder = '-----BEGIN PRIVATE KEY-----' + '\n\n\n\n\n\n' + '-----END PRIVATE KEY-----';
```

Comment:
I think this will be used in other places too. maybe a global const

Reply:
It's being used at one other place. Will make this a constant as part of PR https://github.com/wxsd-sales/roomos-device-widgets/pull/36.